src/components/dashboard/FaceRecognition/FRError.js

```diff
@@ -17,39 +16,45 @@ if (Platform.OS === 'web') {
   Image.prefetch(Oops)
```

Comment:
as i can see, the issue with svg isn't resolved yet

why we couldn't  bundle it to the JS code ? will it take a lot of time ?

Reply:
i'll add task to icebox about bundling svgs